src/Cache/CacheContent.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\View\Cache;
+
+use DateInterval;
+use InvalidArgumentException;
+use Yiisoft\Cache\CacheInterface;
+use Yiisoft\Cache\CacheKeyNormalizer;
+use Yiisoft\Cache\Dependency\Dependency;
+
+use function array_merge;
+use function get_class;
+use function gettype;
+use function is_string;
+use function is_object;
+use function sprintf;
+use function strtr;
+
+/**
+ * CacheContent caches content, supports the use of dynamic content {@see DynamicContent} inside cached content.
+ */
+final class CacheContent
```

Comment:
Cache content sounds like an action, maybe:

```suggestion
final class ContentCache
```

Reply:
Then instead of `DynamicContent` need `ContentDynamic`?

Reply:
No.